react/features/base/react/components/native/AbstractPagedList.js

```diff
@@ -133,19 +152,15 @@ export default class AbstractPagedList extends Component<Props, State> {
     _selectPage(pageIndex: number) {
         const validatedPageIndex = this._validatePageIndex(pageIndex);
 
-        this.setState({
-            pageIndex: validatedPageIndex
-        });
-
-        // The page's Component may have a refresh(dispatch) function which we
-        // invoke when the page is selected.
-        const selectedPage = this.props.pages[validatedPageIndex];
+        const { onSelectPage } = this.props;
 
-        if (selectedPage && selectedPage.component) {
-            const { refresh } = selectedPage.component;
-
-            typeof refresh === 'function' && refresh(this.props.dispatch);
+        if (typeof onSelectPage === 'function') {
+            onSelectPage(validatedPageIndex);
         }
+
+        this.setState({
+            pageIndex: validatedPageIndex
+        }, () => this._maybeRefreshActivePage());
```

Comment:
Well, instead of `() => this._maybeRefreshActivePage()` we usually do `this._maybeRefreshActivePage.bind(this)` in the constructor and then pass this._maybeRefreshActivePage.

Reply:
Seriously, our coding style requires the use of one name per abstraction! Please stop calling it both selected and active page!